.eslintrc.json

```diff
@@ -5,13 +5,29 @@
     "standard",
     "prettier",
     "prettier/standard",
+    "plugin:unicorn/recommended",
     "plugin:ava/recommended",
     "plugin:you-dont-need-lodash-underscore/all"
   ],
   "rules": {
     "no-process-exit": 0,
     "object-shorthand": "error",
     "require-await": "error",
+    "unicorn/custom-error-definition": 2,
+    "unicorn/no-unused-properties": 2,
+    "unicorn/filename-case": [2, { "case": "kebabCase", "ignore": [".*.md"] }],
+    "unicorn/explicit-length-check": [2, { "non-zero": "not-equal" }],
+    "unicorn/no-process-exit": 0,
```

Comment:
We execute `process.exit()` in many files, which is forbidden by [this rule](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-process-exit.md), which recommends throwing an error and running `process.exit()` only in the top-level binary file. This might be a good practice, but this would require too much refactoring at this point.

Reply:
Agree. I'm trying to handle those `process.exit` a few at a time